### PR TITLE
PS-7256 [Doc] Update Binary Tarball Description (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/installation.rst

### CODE REVIEW

**Review by kathy-williamson — APPROVED** [review]

File: doc/source/installation.rst [hunk 1]
```diff
@@ -37,23 +37,26 @@ Following guides describe the installation process for using the official Percon
...
-   * - Percona-Server-5.7.xx-xx-Linux.x86_64.glibc2.12-minimal.tar.gz
+   * - Minimal
+     - Percona-Server-<version number>-Linux.x86_64.glibc2.12-minimal.tar.gz
      - Contains binaries, and libraries but does not include test files, or debug symbols
 
-Both binary tarballs support all distributions.
+Select the |Percona Server| 8.0 version number and type of tarball for your installation. Both binary tarballs support all distributions.
 
-|Percona Server| offers multiple tarballs depending on the *OpenSSL* library available in the distribution:
+In |Percona Server| before 8.0.20-11, multiple tarballs are provided based on the *OpenSSL* library available in the distribution:
```
Comment (EvgeniyPatlan) [inline doc/source/installation.rst]: based on glibc not on openssl version

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: doc/source/installation.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/installation.rst]: Fixed. Thanks.

File: doc/source/installation.rst [hunk 2]
```diff
@@ -37,23 +37,26 @@ Following guides describe the installation process for using the official Percon
...
-   * - Percona-Server-5.7.xx-xx-Linux.x86_64.glibc2.12-minimal.tar.gz
+   * - Minimal
+     - Percona-Server-<version number>-Linux.x86_64.glibc2.12-minimal.tar.gz
      - Contains binaries, and libraries but does not include test files, or debug symbols
 
-Both binary tarballs support all distributions.
+Select the |Percona Server| 8.0 version number and type of tarball for your installation. Both binary tarballs support all distributions.
 
-|Percona Server| offers multiple tarballs depending on the *OpenSSL* library available in the distribution:
+In |Percona Server| before 8.0.20-11, multiple tarballs are provided based on the glibc version available in the distribution:
```
Comment (EvgeniyPatlan) [inline doc/source/installation.rst]: Sorry I was wrong I missed key word ‘before 8.0.20’ yes before 8.0.20 we provided tars based on OpenSSL version. Please revert this change back

Comment (patrickbirch) [reply doc/source/installation.rst]: No problem. Fixed.

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**